processing/src/main/java/org/apache/carbondata/processing/loading/TableProcessingOperations.java

```diff
@@ -53,12 +52,14 @@
   private static final Logger LOGGER =
       LogServiceFactory.getLogService(CarbonLoaderUtil.class.getName());
 
+  private static List<CarbonFile> filesInTrashFolder = new ArrayList<CarbonFile>();
+
   /**
    * delete folder which metadata no exist in tablestatus
    * this method don't check tablestatus history.
    */
   public static void deletePartialLoadDataIfExist(CarbonTable carbonTable,
```

Comment:
this method should move to clean files.
another pr is also changing it.
https://github.com/apache/carbondata/pull/3935
@Pickupolddriver

Reply:
This method is being called from CarbonCleanFIlesCommand class